core/pkg/opencost/allocation.go

```diff
@@ -122,11 +122,12 @@ func (orig LbAllocations) Clone() LbAllocations {
 }
 
 type LbAllocation struct {
-	Service string  `json:"service"`
-	Cost    float64 `json:"cost"`
-	Private bool    `json:"private"`
-	Ip      string  `json:"ip"`    //@bingen:field[version=19]
-	Hours   float64 `json:"hours"` //@bingen:field[version=21]
+	Service    string  `json:"service"`
+	Cost       float64 `json:"cost"`
+	Adjustment float64 `json:"adjustment"` //@bingen:field[ignore]
```

Comment:
Hey @mbolt35 when we add fields to bingen-serialized structs but the new field is ignored, does it still have to be added as the last field in the struct's list of fields? I recall this was a requirement for bingen fields in general but it may not apply to ignored fields.

Reply:
I would move to the end TBH just to avoid any chance of this

Reply:
Yep, good call out. I'll move it just to be safe.

Reply:
Done